life/original/12-chapter-11.adoc

```diff
@@ -186,7 +194,8 @@ and thousands beside myself can bear witness, his life and death was to the prai
 honour and glory of the grace of God; unto whom be the glory of all his works forever,
 Amen.
 
-George Whitehead Says Respecting Him:
+[.blurb]
+=== George Whitehead Says Respecting Him:
```

Comment:
I tried this as an [offset] too, but they both looked equally fine to me, but this is how it looked in the ODT, so I stuck with that.